core/templates/pages/exploration-editor-page/editor-tab/exploration-editor-tab.component.spec.ts

```diff
@@ -558,17 +558,41 @@ describe('Exploration editor tab component', () => {
     expect(component.interactionIsShown).toBe(true);
   });
 
-  it('should save state interaction id', () => {
+  it('should save state interaction data when customization args' +
+      ' are changed', () => {
     stateEditorService.setActiveStateName('First State');
     stateEditorService.setInteraction(
       explorationStatesService.getState('First State').interaction);
 
     expect(stateEditorService.interaction.id).toBe('TextInput');
+    expect(stateEditorService.interaction.customizationArgs).toEqual({
+      rows: { value: 1 },
+      placeholder: { value: new SubtitledUnicode('', 'ca_placeholder') },
+      catchMisspellings: {
+        value: false
+      }
+    });
 
-    let newInteractionId = 'Continue';
-    component.saveInteractionId(newInteractionId);
+    let newInteractionData = {
+      interactionId: 'TextInput',
```

Comment:
This doesn't change from what it was before, so in reality the expectation on line 592 is catching nothing with regards to this field. Is that intended?